File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// Claude SDK utility: result_body

class ClaudeResultBody
{
    public static function call(ClaudeContext $ctx): ?ClaudeResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/rb/utility/result_body.rb
# Claude SDK utility: result_body
module ClaudeUtilities
  ResultBody = ->(ctx) {
    response = ctx.response
    result = ctx.result
    if result && response && response.json_func && response.body
      result.body = response.json_func.call
    end
    result
  }
end
